packages/core/src/external/carequality/ihe-gateway-v2/gateways.ts

```diff
@@ -21,13 +21,18 @@ const specialNamespaceRequiredUrl =
 const pointClickCareOid = "2.16.840.1.113883.3.6448";
 const redoxOid = "2.16.840.1.113883.3.6147.458";
 const redoxGatewayOid = "2.16.840.1.113883.3.6147.458.2";
+const surescriptsOid = "2.16.840.1.113883.3.2054.2.1.1";
 
 /*
  * These gateways only accept a single document reference per request.
  */
-const gatewaysThatAcceptOneDocRefPerRequest = [pointClickCareOid, redoxOid, redoxGatewayOid];
+const gatewaysThatAcceptOneDocRefPerRequest = [
+  pointClickCareOid,
+  redoxOid,
+  redoxGatewayOid,
+  surescriptsOid,
+];
```

Comment:
adding surescripts to this list. 
Context here is surescripts doesnt like it if there are multiple document references being requested with different oids. Simplest thing here is to just sent one document reference at a time